app/controllers/ctc/ctc_pages_controller.rb

```diff
@@ -1,11 +1,6 @@
 module Ctc
   class CtcPagesController < CtcController
     def home
-      if params[:ctc_beta] == "1" && ENV['DISABLE_CTC_BETA_PARAM'].blank?
```

Comment:
I believe this redirect was added because before launch last year, we didn't have links to start intake on the home page "toggling" for when intake is open. Now we can toggle the appearance what links to start intake or sign up for more info show to people, so we can simplify by not having a redirect in the controller.